var/spack/repos/builtin/packages/py-sonata-network-reduction/package.py

```diff
@@ -13,6 +13,7 @@ class PySonataNetworkReduction(PythonPackage):
     git      = "ssh://bbpcode.epfl.ch/nse/sonata-network-reduction"
 
     version('0.0.1', tag='sonata-network-reduction-v0.0.1')
+    version('0.0.2', tag='sonata-network-reduction-v0.0.2')
```

Comment:
Usually we put more recent version to the top.

Reply:
Well, I pushed the changes to my `asanin-epfl:py-sonata-network-reduction0.0.2` branch but they don't appear here.

Reply:
I guess github is slow today :)